### PR TITLE
Change test strings for clearer picture of what's going on

### PR DESCRIPTION
**Pull request by sdhull**
It was unclear to me what scenarios would legitimately render a body that matched "error" vs when "hello world" was rendered because index.html was rendered or because the middleware was rendering that string directly.

This changes index.html to contain the string "Original body" and the middleware to send the string "special error handling" to disambiguate the two.

Additionally it adds some assertions around what the body _should_ match for examples that only had what it _should not_ match.

On a related note: I'm still trying to figure out why index.html is rendered fine with `resilient` enabled in these tests but not in `fastboot-app-server` tests (even after updating fastboot-app-server to properly pass `resilient` through).

### CODE REVIEW

**Review by rwjblue — COMMENTED** [review]
Seems good to me, mind poking at why CI is failing?

**Comment by sdhull** [comment]
@rwjblue ✅

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 language: node_js
 node_js:
   - "stable"
-  - "4"
```
Comment (rwjblue) [inline .travis.yml]: I think we should make this change independently, and also update the `engines` in `package.json`. It'll require a major version bump.

I think we should support 8, 10, and >= 12.

**Comment by sdhull** [comment]
@rwjblue master has been failing since @kratiahuja merged the update to Fastboot 2.x because Fastboot 2.x doesn't support Node 4.x

How would you like to proceed?

**Comment by rwjblue** [comment]
Basically like I mentioned above https://github.com/ember-fastboot/fastboot-express-middleware/pull/48#discussion_r302794801

bump .travis.yml and package.json's engines config in a separate PR (for easier inclusion in the changelog)

**Comment by sdhull** [comment]
OK I can do that. Should I bump the major version as well, considering that [fastboot went from 1.x to 2.x](https://github.com/ember-fastboot/fastboot/blob/master/CHANGELOG.md) as a result of "updating node.js support matrix"

**Comment by rwjblue** [comment]
No, I'll bump major's when I release

**Comment by sdhull** [comment]
@rwjblue I've removed the commit that changes travis yml, but that means CI will fail until we merge #49 

And I assume you're already aware but looks like @kratiahuja has already bumped major version yesterday

**Comment by sdhull** [comment]
@rwjblue I also just opened issue #50 to try to figure out why a couple of these test cases pass when afaict they shouldn't

Also, I'm not sure how these fixtures work... I'd like to spruce up the `rejected-promise` fixture so that the test case can truly demonstrate that it does not render any markup from Ember. Right now it just demonstrates that it doesn't render an error but rather the raw `index.html` file

**Pull request closed without merging**